tests/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.6
+
+LABEL maintainer "RPS <devops@rpsgroup.com>"
+
+RUN mkdir -p /glider-dac-status
+RUN mkdir /glider-dac-status/logs
+COPY /web/ /glider-dac-status/web
+COPY app.py config.yml flask_environments.py manage.py /glider-dac-status/
+COPY status /glider-dac-status/status
+COPY navo /glider-dac-status/navo
+COPY requirements/requirements.txt /requirements.txt
+COPY tests/test_status.py /glider-dac-status/
+RUN rm -f /glider-dac-status/test/test_status.py
```

Comment:
Keep this file around/remove the instruction here as we're going to be updating this test module.